CMakeLists.txt

```diff
@@ -1526,6 +1538,10 @@ endif()
 
 # TODO: - Fast huge math
 
+if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64")
```

Comment:
This was missing and causing us to fail to verify the signature on one of the certs in google.com's chain.